lib/devices/Viomi.js

```diff
@@ -48,10 +64,16 @@ class Viomi extends MiioVacuum {
         /**
          * Current cleaning operation.
          * [mode, pause/start (, opt: roomspec)]
+         *
+         * mode reflects this.mode with 3=zoned clean.
          */
         this.operation = [0, 1, 0];
 
-        /** @type {number} Current operation mode. Probably equal to MopMode with 3=zoned clean. */
+        /**
+         * @type {number} Current operation mode.
+         * 0="normal cleaning"
+         * 1="mop moves" (unsure if only enabled in mop-mode)
+         */
         this.mode = 0;
```

Comment:
While this isn't even part of the PR, this should be refactored to an enum object and referenced by name.

e.g. `MODES.NORMAL_CLEANING`

Not in this PR though

Reply:
Done.